Makefile

```diff
@@ -103,6 +105,12 @@ shellcheck:
 	@echo ""
 	@scripts/shell_check.sh
 
+.PHONY: gofmt_check
+gofmt_check:
```

Comment:
:+1:  Thanks for this! We should probably add `go lint` at some point as well. Though the results aren't exactly pretty right now...

Reply:
Definitely, I thought about adding a ticket to address it during Hacktoberfest, but realized there were way too many warnings to ask anyone from the community to work on fixing it.